automation/features/step-definitions/staking.steps.js

```diff
@@ -1,132 +1,179 @@
+});
+
+Then(/^the stake is successful$/, () => {
+    $('.callout--success').waitForDisplayed({ timeout: 60000, timeoutMsg: "stake success message container not displayed after 60 seconds" })
+    expect($('.callout__title=At the beginning of the next epoch your $VEGA will be nominated to the validator')).toBeDisplayed()
+});
+
+Then(/^I click on the back to staking button$/, () => {
+    stakingPage.backToStakingPageBtn.click()
+});
+
+Then(/^I am back on the staking main page$/, () => {
+    expect(browser.getUrl()).toEqual(`${browser.options.baseUrl}/staking`)
+});
+
+
 When(/^I click the pending transactions button$/, () => {
     browser.getByTestId('pending-transactions-btn').click()
 });
 
 Then(/^I can see the pending transactions modal is shown$/, () => {
- browser.getByTestId('pending-transactions-modal').waitForDisplayed({timeout:20000,timeoutMsg:"pending transactions modal did not display"})
-expect(stakingPage.etherscanLink).toBeDisplayed()
-expect(stakingPage.etherscanLink).toHaveAttr('href')
-expect(stakingPage.pendingTransactionsModalStatus).toHaveText('Pending')
+    browser.getByTestId('pending-transactions-modal').waitForDisplayed({ timeout: 20000, timeoutMsg: "pending transactions modal did not display" })
+    expect(stakingPage.etherscanLink).toBeDisplayed()
+    expect(stakingPage.etherscanLink).toHaveAttr('href')
+    expect(stakingPage.pendingTransactionsModalStatus).toHaveText('Pending')
+});
+
+When(/^I can see "([^"]*)?" vega has been removed from staking$/, (tokenAmount) => {
+    $('.callout--success').waitForDisplayed({ timeout: 60000, timeoutMsg: "remove stake success message container not displayed after 60 seconds" })
+    expect($(`.callout__title*=${tokenAmount} $VEGA has been removed from validator`)).toBeDisplayed()
+    expect($('p=It will be applied in the next epoch')).toBeDisplayed()
+});
+
+When(/^I click on the option to remove stake now$/, () => {
+    stakingPage.removeStakeNowBtn.waitForDisplayed({ timeout: 20000, timeoutMsg: "remove stake now button not displayed" })
+    stakingPage.removeStakeNowBtn.click()
+});
+
+When(/^I can see the remove now disclaimer with text "([^"]*)?"$/, (disclaimerText) => {
+    stakingPage.removeStakeNowDisclaimerText.waitForDisplayed({ timeout: 20000, timeoutMsg: "remove stake disclaimer text not displayed" })
+    expect(stakingPage.removeStakeNowDisclaimerText.getText()).toEqual(disclaimerText)
+});
+
+Then(/^the submit button text is "([^"]*)?"$/, (btnText) => {
+    stakingPage.tokenAmountSubmitBtn.waitForEnabled()
+    expect(stakingPage.tokenAmountSubmitBtn).toHaveText(btnText)
+});
+
+Then(/^I can see the stake is removed immediately$/, () => {
+    $('.callout--success').waitForDisplayed({ timeout: 60000, timeoutMsg: "remove stake now success message container not displayed after 60 seconds" })
```

Comment:
There's quite a few steps here with timeouts of 20-60 seconds. Is it possible to reduce these? 
If anything takes more than 10 seconds to load that should probably be looked at. Understandable if some of these are waiting for epochs etc.

Reply:
this particulat step is waiting for the success of stake removal so might actually need to be longer , but we wont be able to do much anbout reducing these to 10 seconds until we get capsule installed